library.c

```diff
@@ -1575,6 +1794,96 @@ redis_xinfo_reply(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock, zval *z_t
     return FAILURE;
 }
 
+PHP_REDIS_API int
+redis_read_acl_log_reply(RedisSock *redis_sock, zval *zret, long count) {
+    zval zsub;
+    int i, nsub;
+
+    for (i = 0; i < count; i++) {
+        if (read_mbulk_header(redis_sock, &nsub) < 0 || nsub % 2 != 0)
+            return FAILURE;
+
+        array_init(&zsub);
+        if (redis_mbulk_reply_zipped_raw_variant(redis_sock, &zsub, nsub) == FAILURE)
+            return FAILURE;
+
+        add_next_index_zval(zret, &zsub);
+    }
+
+    return SUCCESS;
+}
+
+PHP_REDIS_API int
+redis_read_acl_getuser_reply(RedisSock *redis_sock, zval *zret, long count) {
+    REDIS_REPLY_TYPE type;
+    zval zv;
+    char *key, *val;
+    long vlen;
+    int klen, i;
+
+    for (i = 0; i < count; i += 2) {
+        if (!(key = redis_sock_read(redis_sock, &klen)) ||
+            redis_read_reply_type(redis_sock, &type, &vlen) < 0 ||
+            (type != TYPE_BULK && type != TYPE_MULTIBULK) ||
+            klen > INT_MAX || vlen > INT_MAX)
```

Comment:
`klen` can't be greater than `INT_MAX` since it's an `int`.

Reply:
Good catch! :man_facepalming: